kickstarts/base.ks.erb

```diff
@@ -1,5 +1,8 @@
 # Kickstart file automatically generated by anaconda.
-# Run the text based installer to see dependency info in screenshots
+
+# By default run the text based installer to see dependency info in screenshots
+<% if ENV['KS_GUI'] == '1' then "" else "text" end %>
+
```

Comment:
@geertj Thanks, that makes sense.  I'm good with this.  How much does this impact the image size though?